### PR TITLE
test: fix nightly nas and iris tf keras tests [DET-3264]

### PR DESCRIPTION
**Pull request by aaron276h**
## Description
NAS was using the old form of gradient clipping. Iris test needed to have a random seed set to reduce flakiness.

## Test Plan
Tested locally.

### CODE REVIEW

**Comment by aaron276h** [comment]
@yoavz Yeah i'd agree, just handle them as they come up.
